src/base/h5_compression_argument.yaml

```diff
@@ -0,0 +1,9 @@
+arguments:
+  - name: "--output_compression"
+    description: |
+      Compression format to use for the output AnnData and/or Mudata objects.
+      By default no compression is applied.
+    type: string
+    choices: ["gzip", "lzf"]
+    required: false
+    example: "gzip"
```

Comment:
Add newline

```suggestion
    example: "gzip"

```


Also, should we set the default to gzip?


```suggestion
    default: "gzip"

```

If we do, we should add "none" to the list of choices so we can unset it while viash-io/viash#762 is still not implemented...

Reply:
It was previously proposed to set the default to `gzip`, but that slows the pipelines down because all intermediary steps needs to do the compression while in theory its only really useful to be enabled on the last step.